modules/ROOT/pages/database-internals/store-formats.adoc

```diff
@@ -339,7 +442,7 @@ The following tables show the format and Neo4j version compatibility and the lim
 | `2^35` (34 359 738 368)
 |===
 
-[role=enterprise-edition]
+[role=enterprise-edition label--deprecated-5.23]
```

Comment:
This doesn't seem to get rendered. It would be good to have a visible tag here and a link to the deprecation section.